### DISCUSSION TITLE
Can't find Script Canvas Function

### DISCUSSION THREAD

**Discussion by soltsmank** [o3de/o3de#6848] [Q&A]
Hello O3DEneers.

Following this Guide [ScriptCanvas functions](https://o3de.org/docs/user-guide/scripting/script-canvas/editor-reference/functions/) , I don't see any icon or main menu to create a new Function.
I double-check Gems, found nothing about Functions.

How to enable it?
Is it deprecated?

Thanks

**Comment by FiniteStateGit** [comment]
Yeah this one got me too, you have to click on these the icons to add the input and output nodes.

![create-sc-function](https://user-images.githubusercontent.com/31811688/149269276-4b14448b-1230-45da-89ef-58501a4fa5dc.png)
